### PR TITLE
Problem7 has been solved

### PR DESCRIPTION
**Pull request by ogulcanakkir**

### CODE REVIEW

File: dotnet/src/Problem7.cs [hunk 1]
```diff
@@ -6,41 +6,39 @@ namespace ProjectEuler
 {
     public static class Problem7
     {
-        public static int bulunan = 0, son = 10001;
-        public static List<int> asalsayilar = new List<int>();
+        public static int CatchPrimeNumber = 0, LastPrimeNumber = 10001;
+        public static List<int> PrimeNumbers = new List<int>();
```
Comment (UmutComlekci) [inline dotnet/src/Problem7.cs]: Why are you store the prime numbers in array?

File: dotnet/src/Problem16.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Numerics;
+using System.Linq;
+
+namespace ProjectEuler
+{
+    /*
+    * 215 = 32768 and the sum of its digits is 3 + 2 + 7 + 6 + 8 = 26.
```
Comment (UmutComlekci) [inline dotnet/src/Problem16.cs]: To write exponential value, you can write 2^15

File: dotnet/src/Problem16.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    * What is the sum of the digits of the number 2^1000?
+    */
+
+    public static class Problem16
+    {
+        public static double Sum = 0;
```
Comment (UmutComlekci) [inline dotnet/src/Problem16.cs]: We don't recommend to use global variables like this. Move your variable into method scope

Comment (ogulcanakkir) [reply dotnet/src/Problem16.cs]: Thank you for your advice. I will be more careful after that.

**Pull request merged**